unit-tests/live/d400/test-hdr.py

```diff
@@ -0,0 +1,638 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+######################################
+# This set of tests is valid for any device that supports the HDR feature #
+######################################
+
+# test:device D400* !D457
```

Comment:
Can we also try it on D457?
I think it should work

Reply:
It should - please try

Reply:
needed to do some adjustments to make it work, but yes seems like it's working for it now

Reply:
great! Well done Avia!